async listed as a 'dev dependency'

async is listed as a 'dev dependency', so trying to run `npm run download` after `npm install --production` results in `Error: Cannot find module 'async'`.

it might be better to move this (and any other associated dependencies) in to the main dependency list if they are intended to be used in production.

--- a/package.json
+++ b/package.json
@@ -39,6 +39,7 @@
     "elasticsearch": ">=1.3.4"
   },
   "dependencies": {
+    "async": "^2.0.0",
     "combined-stream": "^1.0.5",
     "extend": "^3.0.0",
     "fs-extra": "^4.0.0",
@@ -61,7 +62,6 @@
     "trimmer": "^2.0.0"
   },
   "devDependencies": {
-    "async": "^2.0.0",
     "colors": "^1.1.2",
     "deep-diff": "^0.3.2",
     "event-stream": "^3.3.2",

